integration_tests/tests/temporary-accommodation/manage/booking.cy.ts

```diff
@@ -143,8 +142,12 @@ context('Booking', () => {
       expect(requests).to.have.length(1)
       const requestBody = JSON.parse(requests[0].body)
 
-      expect(requestBody.service).equal('temporary-accommodation')
-      expect(requestBody.bedId).equal(bedspace.id)
+      if (Cypress.env('ENABLE_CAS3V2_API') === true) {
```

Comment:
Cypress doesn't like to access `config.flags` when running the integration tests, so instead we have to rely on this extra Cypress environment variable. The Github Actions workflow specifies this as `CYPRESS_ENABLE_CAS3V2_API`. The extra workflow will be fully removed when this is deployed.